jupyterhub_moss/form/option_form.js

```diff
@@ -208,7 +211,7 @@ function restoreCustomEnvironmentsFromLocalStorage() {
   }
 
   for (const key in config) {
-    addCustomEnvironment(key, config[key].description, config[key].path, false);
+    addCustomEnvironment(key, config[key].description, config[key].path, config[key].modules, false);
```

Comment:
This needs to support local storage created before this change (where `config[key].modules` is `undefined`):
```suggestion
    addCustomEnvironment(key, config[key].description, config[key].path || "", config[key].modules || "", false);
```
Alternative is to bump the version of the `custom-environment-config` storage, but not worth it IMO.